runner/app/live/pipelines/loader.py

```diff
@@ -1,10 +1,10 @@
 from .interface import Pipeline
 
-def load_pipeline(name: str, **params) -> Pipeline:
+def load_pipeline(name: str) -> Pipeline:
     if name == "comfyui":
         from .comfyui import ComfyUI
-        return ComfyUI(**params)
```

Comment:
`**params` are removed because they are passed into the pipeline after initialization from `process.py` using `pipeline.update_params` https://github.com/livepeer/ai-runner/pull/479/files?diff=unified&w=1#diff-884ec47d03a7fdce1ff7a0fc4f9858f45965ff1cbb3ee034c1f3505f756a7f2bR206.